Samples/WorkflowCombineSampleApp/WorkflowCombineSampleApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,570 @@
+// !$*UTF8*$!
```

Comment:
We should probably not check-in our project files.

Reply:
I've moved the sample app generation into `Development.podspec` instead